src/frontends/pytorch/src/transforms/u4_block_repack.cpp

```diff
@@ -85,7 +85,8 @@ U4BlockRepack::U4BlockRepack() {
                 }
             }
 
-            copy_runtime_info(NodeVector{constant, reshape1, transpose, reshape2}, new_const);
+            copy_runtime_info({std::move(constant), std::move(reshape1), std::move(transpose), std::move(reshape2)},
```

Comment:
Breaks GPTQ int4 models. Should be reverted. @andrei-kochin

Reply:
@rkazants FYI